### PR TITLE
Update upload_ansible_roles_to_galaxy.py script

### PR DESCRIPTION
**Pull request by redhatrises**
- Only update readme and meta when the repo is new. Readme and metadata
  will drift from the template to meet new Galaxy 3 ansible formatting
  requirements as well as line up correct repo tagging.
- Use ComplianceAsCode GH site for homepage so that users know
  where to go for ComplianceAsCode issues and content.

### CODE REVIEW

File: utils/upload_ansible_roles_to_galaxy.py [hunk 1]
```diff
@@ -381,6 +383,7 @@ def main():
     # Create empty repositories
     github_new_repos = sorted(list(set(selected_roles) - set(github_repositories)))
     if github_new_repos:
+        repo_status = "new"
```
Comment (yuumasato) [inline utils/upload_ansible_roles_to_galaxy.py]: If there is a single new repo, won't all repos be update as `new`, and possibly have their readme and content updated?

Comment (redhatrises) [reply utils/upload_ansible_roles_to_galaxy.py]: That's a good point @yuumasato I didn't think about that. Thanks! I will look into it.

Comment (redhatrises) [reply utils/upload_ansible_roles_to_galaxy.py]: @yuumasato fixed.

**Comment by redhatrises** [comment]
@yuumasato found some other errors and fixed them too.

File: utils/upload_ansible_roles_to_galaxy.py [hunk 2]
```diff
@@ -354,6 +361,7 @@ def locally_clone_and_init_repositories(organization, repo_list):
 
 def main():
     args = parse_args()
+    repo_status = "update"
```
Comment (yuumasato) [inline utils/upload_ansible_roles_to_galaxy.py]: Leftover from previous commit

File: utils/upload_ansible_roles_to_galaxy.py [hunk 3]
```diff
@@ -229,7 +229,8 @@ def _update_vars_content_if_needed(self):
         vars_local_content = yaml.dump(self.vars_data, width=120, indent=4,
                                        default_flow_style=False)
 
-        if vars_local_content != vars_remote_content.decoded_content:
+        if vars_local_content != vars_remote_content.decoded_content and \
+            vars_local_content.splitlines()[0] != "null":
```
Comment (yuumasato) [inline utils/upload_ansible_roles_to_galaxy.py]: This needs pep8 indentation fix.

File: utils/upload_ansible_roles_to_galaxy.py (hunk 2 above)
Comment (redhatrises) [reply utils/upload_ansible_roles_to_galaxy.py]: @yuumasato thanks fixed.

File: utils/upload_ansible_roles_to_galaxy.py (hunk 3 above)
Comment (redhatrises) [reply utils/upload_ansible_roles_to_galaxy.py]: @yuumasato thanks fixed.

**Pull request merged**